recipes/aruco/3.x.x/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class LibnameConan(ConanFile):
+    name = "aruco"
+    description = "Augmented reality library based on OpenCV "
+    topics = ("conan", "aruco", "augmented reality")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.uco.es/investiga/grupos/ava/node/26"
+    license = "GPL-3.0-only"
+    no_copy_source = True
+    generators = "cmake"
+    _cmake = None
+
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [False, True], "fPIC": [False, True]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def requirements(self):
+        self.requires('opencv/4.2.0@conan/stable')
```

Comment:
```suggestion
        self.requires('opencv/4.2.0@conan/stable')
```

Conan Community and Bincrafters packages are not allowed in CCI. The only OpenCV version available on CCI is 2.4. We should have 3.x and 4.x in the future.

Reply:
Is someone currently working on openCV 4.x? I tested and the Aruco library does not compile with openCV 2.4.

Reply:
Getting https://github.com/conan-io/conan-center-index/pull/3409 in the repo will unblock this.